app/Services/GocardlessMapper.php

```
@@ -60,47 +59,207 @@
+
+        return implode(' ', array_filter($parts));
+    }
+
+    /**
+     * Extracts the partner name from the transaction data.
      *
-     * Extracts and normalizes transaction details such as account identifiers, IBANs, amounts, currency, booking and processing dates, partner information, description, transaction type, balance after transaction, metadata, and includes the original transaction data.
+     * @param array $transaction The transaction data.
+     * @return string|null The partner name or null if not found.
+     */
+    private function extractPartnerName(array $transaction): ?string
```

PHPStan (2.1.15)

Method App\Services\GocardlessMapper::extractPartnerName() has parameter $transaction with no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)